docs/rfc/rfc-1.adoc

```diff
@@ -283,20 +283,21 @@ it.
 
 Governable Parameters:
 
-- `failed_heartbeat_sortition_timeout`: The amount of time an operator is
-  removed from the sortition pool after failing a heartbeat.
+- `failed_heartbeat_reward_timeout`: The amount of time an operator is
```

Comment:
Is `_timeout` correct word? I am not going to argue with a native speaker 😁 but after reading it the first time, I thought this parameter is about a timeout to provide a heartbeat. Isn't it more something like a... reward ban period? 

This parameter should say for how long the operator does not receive future rewards although they are in the sortition pool.

Reply:
hah, please argue! 

There are only two hard things in Computer Science: cache invalidation and naming things.

-- Phil Karlton

In my head, I was thinking about when my mom would put me in "time out" when I was bad 👿 . Bad naming; ambiguous, I'll put myself in time out.

address in 8cce32e